doc/docs/reference/patient_list.md

```diff
@@ -1,10 +1,9 @@
-# `opal.core.patient_lists`
+## opal.core.patient_lists
 
 The `patient_lists` module defines a number of classes for working with lists of patients.
 
-## `class  PatientList`
 
-### Properties and Class Methods
+### class opal.core.patient_lists.PatientList
```

Comment:
if we're doing this: `opal.core.patient_lists.PatientList`, shouldn't we do the same in `TabbedPatientListGroup` and `TaggedPatientList`?

Is it necessary though? it says `opal.core.patient_lists` at the top of the page.